ajax/libs/slippry/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "slippry",
+  "filename": "slippry.min.js",
+  "version": "1.4.0",
+  "description": "Responsive slider plugin for jQuery. Intrinsic trick to enable responsive resizing without js.",
+  "keywords": [
+    "jQuery",
+    "Responsive",
+    "slider",
+    "plugin",
+    "PIXELS"
+  ],
+  "author": "Lukas Jakob Hafner <luki@booncon.com>",
+  "homepage": "http://slippry.com",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/booncon/slippry.git"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/booncon/slippry.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      },
+      {
+        "basePath": "src",
+        "files": [
+          "*.+(css|js|map)"
+        ]
+      },
+      {
+        "basePath": "images",
```

Comment:
@pvnr0082t In this case, I set dist and src as basePath, the best way I can think of is also setting images as basePath to include the files under images directory. Could you give me some advice about it? Thank you.